CASMTRIAGE-6643 `install-ncn-bond-members` note (#4714)
Cray-HPE/docs-csm: install/Create_System_Configuration_Using_SHCD.md

1. (`pit#`) Edit the `system_config.yaml` file with the appropriate values.

   > ***NOTES***
   >
   > - For a short description of each key in the file, run `csi config init --help`.

>    >   ***IMPORTANT*** `install-ncn-bond-members` have many possibilities, but are typically:
>    >      - `p1p1,p10p1` for HPE nodes
>    >      - `p1p1,p1p2` for Gigabyte nodes
>    >      - `p801p1,p801p2` for Intel nodes

Reviewer: ```suggestion
   >   ***IMPORTANT*** `install-ncn-bond-members` have many possibilities but are typically:
   >      - `p1p1,p10p1` for HPE nodes.
   >      - `p1p1,p1p2` for Gigabyte nodes.
   >      - `p801p1,p801p2` for Intel nodes.
```

>    > - For more description of these settings and the default values, see

Reviewer: ```suggestion
   > - For more details on these settings and the default values, see
```
